CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
-# Changelog
+# Change Log - @splunk/otel
+
+This log was last generated on Tue, 17 Aug 2021 16:14:33 GMT and should not be manually modified.
```

Comment:
Is this generated from changelog.json or from git? Do we have the ability to only add user facing changes to the log and skip internal stuff such refactoring commits?

Reply:
It's based on files like <https://github.com/signalfx/splunk-otel-js/blob/3f8bb388bcd96fff93f66447c8d46316434c415f/change/%40splunk-otel-5756ef5a-6dc7-43e9-be09-e4766af9fcf1.json>, currently there's a check (in linter job) if at least one change was added in each PR. I think we can either get rid of that check or use an issue type which will be filtered out when generating a release. Git commit messages are not used in the process. So I looked into it, and <https://github.com/googleapis/release-please> would be better, and it uses git commits, but it requires a Github bot, so it's sort of incompatible with our infrastructure.